.drone.star

```diff
@@ -53,7 +53,7 @@ config = {
 			},
 			'servers': [
 				'daily-master-qa',
-				'latest'
+				// 'latest'
```

Comment:
There is another "latest" further down.

And, IMO, "latest" is in the default "servers" list. So actually "daily-master-qa"-only will need to be specified in each acceptance tests section.

Reply:
`.drone.star:56:7: got '//', want primary expression `
https://drone.owncloud.com/owncloud/password_policy/1846

`#` is a comment in starlark.